### DISCUSSION TITLE
Allow translations to be shown side-by-side by default

### DISCUSSION THREAD

**Discussion by igorricardo20** [directus/directus#20270] [Feature Requests]
### Summary

Currently, when we open an item within a collection, only one translation is opened by default, there is an option to choose the default language or be based on the user language. But I haven't found an option to always show 2 languages. I think it would add more user experience since the filling of contents will be slightly faster 

### Basic Example

Instead of only showing Dutch when opening the collection, it could show automatically Dutch and English (without the necessity to click in the side-by-side view) 

### Motivation

When filling many fields in a row, it makes the process a bit easier. 

### Detailed Design

Maybe a toggle in the field display area enabling the side-by-side view by default

### Drawbacks

If this is something easy to do. I think is worth it since I have received this suggestion from 2 people already

### Alternatives

Maybe, allowing to "save" the field view state, not just for the translations but all others 

### Adoption Strategy

This would be almost seamless since it's just a matter of customizing the views (could be per user so not every user would notice the difference if we don't want them to see...)

**Comment by kolosseo** [comment]
Seems a duplicate of
 #https://github.com/directus/directus/discussions/15458

That would be a very nice feature, I add it would be also very nice to have the possibility to split in more than just 2 columns.

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
